tests/profiling/test_accuracy.py

```diff
@@ -61,7 +61,7 @@ def spend_cpu_3():
 
 
 def almost_equal(value, target, tolerance=TOLERANCE):
-    return target * (1 + tolerance) >= value >= target * (1 - tolerance)
+    return abs(value - target) / target <= tolerance
```

Comment:
I somehow had to change this to make the accuracy tests pass. We can get back to this later and see if it's really required. On a theoretical level, I'd say this would be a better implementation anyway, so worth making this change regardless.